src/Functional.CQS.AOP.CommonTestInfrastructure/DummyObjects/DummyAsyncCommandHandlerThatFails.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Functional.CQS.AOP.CommonTestInfrastructure.DummyObjects
+{
+	/// <summary>
+	/// Sample <see cref="IAsyncCommandHandler{TCommand, TError}"/> implementation.  Returns a failure result.
+	/// </summary>
+	public class DummyAsyncCommandHandlerThatFails : IAsyncCommandHandler<DummyAsyncCommandThatFails, DummyAsyncCommandError>
+	{
+		private static readonly DummyAsyncCommandThatFails _command = new DummyAsyncCommandThatFails();
```

Comment:
`_command` doesn't appear to be used.

Reply:
Same comment goes for `DummyAsyncCommandHandlerThatSucceeds`.

Reply:
Similar comments go for the other dummy handlers.

Reply:
I've cleaned this all up